tutorials/create-a-usb-stick-on-ubuntu/create-a-usb-stick-on-ubuntu.md

```diff
@@ -1,60 +1,86 @@
 ---
 id: tutorial-create-a-usb-stick-on-ubuntu
-summary: How to write a USB stick with Ubuntu.
+summary: Use your Ubuntu desktop to create a bootable USB stick that can be used to run and install Ubuntu on any USB-equipped PC.
 categories: desktop
 tags: tutorial,installation,usb,ubuntu,desktop
 difficulty: 2
 status: Published
-published: 2017-05-31
+published: 2017-07-11
+author: Canonical Web Team <webteam@canonical.com>
 
 ---
 
 # Create a bootable USB stick on Ubuntu
 
-## Getting started
-Duration: 0:05
+## Overview
+Duration: 0:01
+
+With a bootable Ubuntu USB stick, you can:
 
-To create a USB stick from which you can install Ubuntu, you must first [download Ubuntu](https://www.ubuntu.com/download) and, if you want, [verify the download](/tutorial/tutorial-how-to-verify-ubuntu).
+- Install or upgrade Ubuntu.
+- Test out the Ubuntu desktop experience without touching your PC configuration.
+- Boot into Ubuntu on a borrowed machine or from an internet cafe.
+- Use tools installed by default on the USB stick to repair or fix a broken configuration.
 
-Then, follow these instructions:
+Creating a bootable Ubuntu USB stick is very simple, especially from Ubuntu itself, and we're going to cover the process in the next few steps.
 
-1. Insert a USB stick with at least 2GB of free space
-2. Open the dash and search for Startup Disk Creator
-3. Select the Startup Disk Creator to launch the app
+Alternatively, we also have tutorials to help you create a bootable USB stick from both [Microsoft Windows](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows) and [Apple macOS](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos).
```

Comment:
yep, sure. I was going to bring this up as I'd seen you did this, but we'd going to need to change the other tutorials too (perhaps as we touch them).